libyul/backends/evm/EVMDialect.cpp

```diff
@@ -226,6 +222,18 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 
 }
 
+optional<AbstractAssembly::SubID> BuiltinContext::findSubIDForDataName(YulString const& _dataName)
+{
+	if (subIDs.count(_dataName) == 1)
+		return subIDs.at(_dataName);
+
+	size_t dotPos = _dataName.str().find_last_of('.');
+	YulString subObjectName = dotPos == string::npos ? _dataName : YulString{_dataName.str().substr(dotPos + 1)};
```

Comment:
https://github.com/ethereum/solidity/pull/9047#discussion_r431819599

I find expressions like this easier to parse visually if there's a parenthesis around the condition:
```suggestion
	YulString subObjectName = (dotPos == string::npos ? _dataName : YulString{_dataName.str().substr(dotPos + 1)});
```